src/explore-education-statistics-common/src/polyfill.js

```diff
@@ -52,3 +52,5 @@ if (!Element.prototype.closest) {
     return null;
   };
 }
+
+require('intersection-observer');
```

Comment:
Wrap this with a `!window.IntersectionObserver` check so that we don't require it into the bundle unless the user actually needs it.

Reply:
It has it already in the code, I checked the library

Reply:
Yes, but we can avoid it being pulled into the bundle entirely by doing the check here as well.

Reply:
but whatever